index.html

```diff
@@ -370,20 +382,24 @@ <h3>Use Cases and Requirements</h3>
         </li>
         <li>
 <a>Holders</a> are be able to freely choose and change the agents they employ
-to help them manage and share their <a>verifiable credentials</a>.
+to help them manage their <a>verifiable credentials</a> and generate and share <a>verifiable presentations</a>.
         </li>
         <li>
-<a>Holders</a> that share <a>verifiable credentials</a> are not required to
+<a>Holders</a> that share <a>verifiable presentations</a> never need to
 reveal the identity of the <a>verifier</a> to <a>issuers</a>.
         </li>
         <li>
-A <a>verifiable credential</a> is expressed in one or more standard,
+<a>Holders</a> in <a>presentations</a> can either disclose the attributes or satisfy <a>predicates</a>
+requested by the <a>verifier</a>. <a>Predicates</a> are boolean conditions like greater/less than or
```

Comment:
Also, this section is specific to selective disclosure schemes. We should cover other cases in the language and make clear this is just one option. "Some schemes may support selective disclosure where..."

Reply:
Changed the language.